packages/analytics/src/pingback.ts

```diff
@@ -18,7 +18,7 @@ function sendPingbacks() {
 
 const debouncedPingbackEvent = debounce(1000, sendPingbacks)
 
-const pingback = ({ gif, userId, eventType, actionType, attributes, queueEvents = true }: Pingback) => {
+const pingback = ({ gif, userId, eventType, actionType, attributes, queueEvents = true, arp }: Pingback) => {
```

Comment:
@dleitee maybe we rename this param to `analyticsResponsePayload`, it's for the `Pingback` type, and we leave `analytics_response_payload` of course for `PingbackEvent`, which has a bunch of underscore props already